portal/static/js/src/profile.js

```diff
@@ -1904,7 +1904,13 @@ export default (function() {
                         }
                         item.lastUpdated = self.modules.tnthDates.formatDateString(item.lastUpdated, "iso");
                         item.comment = item.comment ? self.escapeHtml(item.comment) : "";
-                        var c = decodeURIComponent(String(item.comment));
+                        var c = String(item.comment);
+                        try {
+                            c = decodeURIComponent(c);
+                        } catch(e) { //catch error, output to console e.g. URI malformed error. output error to console for debugging purpose
+                            console.log("Error decoding auditing comment " + c);
```

Comment:
I like the more robust decode attempt, but worry about relying on the console.  Should the failure case be transmitted back to the portal's error page?

Reply:
thanks, paul, added fix to report error back to server, so at least we are aware of the error.